terraform-aws-github-runner/modules/runners/scale-up-chron.tf

```diff
@@ -0,0 +1,163 @@
+resource "aws_kms_grant" "scale_up_chron" {
```

Comment:
you want to add a `aws_cloudwatch_event_rule` so you have a trigger for cron execution. Otherwise this lambda would never be called :)